docs/management/cases/add-connectors.asciidoc

```diff
@@ -0,0 +1,57 @@
+[[add-case-connectors]]
+== Add connectors
+
+preview::[]
+
+You can add connectors to cases to push information to these external incident
+management systems:
+
+* IBM Resilient
+* Jira
+* ServiceNow ITSM
+* ServiceNow SecOps
+* {swimlane}
+
+NOTE: To create connectors and send cases to external systems, you must have the
+appropriate {kib} feature privileges. Refer to <<setup-cases>>.
+
+[discrete]
+[[create-case-connectors]]
+== Create connectors
+
+You can create connectors in *Management > {stack-manage-app} > {rules-ui}*, as
+described in <<action-types>>. Alternatively, you can create them in
+*Management > {stack-manage-app} > Cases*:
+
+. Click *Edit external connection*. Alternatively, select a case then click
+*Add connector*.
```

Comment:
I think we should remove the "Alternatively, select a case then click
*Add connector*." because you cannot create a connector from a case. Only from the configuration page, you can do it.